External basemap

csobier opened the pull request:
This branch needs to be reviewed.  Also, I am having markdown difficulties getting the last numbered bullet to appear correctly. Argh!  (Step 3- Click *ADD BASEMAPS*)

andrewxhill commented:
yep, the issue here is that markdown doesn't like to have lists split up... so

1. one
2. two
3. three

is different than,

1. one
2. two

something

3. three

they are a known pain... http://meta.stackexchange.com/questions/137077/cant-start-a-numbered-list-on-a-number-other-than-1

csobier commented:
Updated branch and removed part of procedure, since markdown doesn't support the format I was trying to apply.

@ztephm Can you do a quick review on this branch?

andrewxhill commented:
this is pretty close, there are still some images we will need to go back to cleanup but i'd prefer to get the branch merged and start opening tickets for image issues independently
